demos/cdn-react/CKEditorCloudPluginsDemo.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import React, { type ReactNode } from 'react';
+
+import type { Plugin } from 'https://cdn.ckeditor.com/typings/ckeditor5.d.ts';
+
+import { getCKCdnClassicEditor } from './getCKCdnClassicEditor.js';
+import { CKEditor, useCKEditorCloud } from '../../src/index.js';
+
+type CKEditorCloudPluginsDemoProps = {
+	content: string;
+};
+
+declare global {
+	interface Window {
+		'@wiris/mathtype-ckeditor5': typeof Plugin;
+	}
+}
+
+export const CKEditorCloudPluginsDemo = ( { content }: CKEditorCloudPluginsDemoProps ): ReactNode => {
```

Comment:
Currently Cloud Plugins Demo returns some kind of error.
Please make sure it is fixed before the merge.
![image](https://github.com/user-attachments/assets/043eebf2-2bd0-456b-a080-001633488c1a)

Reply:
Related to https://github.com/wiris/html-integrations/pull/998. After Wirs plugin update, it'll start to work normally.